src/module.c

```diff
@@ -269,27 +275,6 @@ static void finalize_contact(napi_env env, void* data, void* hint) {
   }
 }
 
-static void finalize_context(napi_env env, void* data, void* hint) {
-  if (data) {
-    dcn_context_t* dcn_context = (dcn_context_t*)data;
-    dcn_context->gc = 1;
-    TRACE("cleaning up dc_context..");
-    dc_context_unref(dcn_context->dc_context);
-    TRACE("done cleaning up dc_context");
-    dcn_context->dc_context = NULL;
-
-    strtable_unref(dcn_context->strtable);
-    dcn_context->strtable = NULL;
-
-    pthread_cond_destroy(&dcn_context->dc_event_http_cond);
-    pthread_mutex_destroy(&dcn_context->dc_event_http_mutex);
-
-    TRACE("done");
-
-    free(dcn_context);
-  }
-}
```

Comment:
This means we no longer clean up the context using v8 garbage collection. This is all moved to `dcn_close()`.